src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,40 @@
+.reviews {
```

Comment:
It should be just an element of card block and in HTML you should use it like BEM mix.

Reply:
done. check again, please